platform/darwin/src/MGLOfflinePack.mm

```diff
@@ -78,7 +87,17 @@ - (void)dealloc {
 
     const mbgl::OfflineRegionDefinition &regionDefinition = _mbglOfflineRegion->getDefinition();
     NSAssert([MGLTilePyramidOfflineRegion conformsToProtocol:@protocol(MGLOfflineRegion_Private)], @"MGLTilePyramidOfflineRegion should conform to MGLOfflineRegion_Private.");
-    return [(id <MGLOfflineRegion_Private>)[MGLTilePyramidOfflineRegion alloc] initWithOfflineRegionDefinition:regionDefinition];
+    NSAssert([MGLShapeOfflineRegion conformsToProtocol:@protocol(MGLOfflineRegion_Private)], @"MGLGeometryOfflineRegion should conform to MGLOfflineRegion_Private.");
```

Comment:
Replace all instances of `MGLGeometryOfflineRegion` in this PR with `MGLShapeOfflineRegion`.